tsk/auto/auto_db.cpp

```diff
@@ -208,13 +209,38 @@ uint8_t
 TskAutoDb::addImageDetails(const char* deviceId)
 {
    string md5 = "";
+   string sha1 = "";
+   string collectionDetails = "";
 #if HAVE_LIBEWF 
    if (m_img_info->itype == TSK_IMG_TYPE_EWF_EWF) {
      // @@@ This should really probably be inside of a tsk_img_ method
        IMG_EWF_INFO *ewf_info = (IMG_EWF_INFO *)m_img_info;
        if (ewf_info->md5hash_isset) {
            md5 = ewf_info->md5hash;
        }
+	   if (ewf_info->sha1hash_isset) {
+		   sha1 = ewf_info->sha1hash;
+	   }
+
+       //Need 1MB for libewf read and extra 100 bytes for header name and formatting
+       const size_t buffer_size = 1024100;
+       char result[buffer_size];
```

Comment:
Looking at this again, 1024100 might be a bit large for the stack. I'm not having problems but it might be safer to do a malloc here.